cli/cmd/root.go

```diff
@@ -63,6 +64,7 @@ var RootCmd = &cobra.Command{
 func init() {
 	RootCmd.PersistentFlags().StringVarP(&controlPlaneNamespace, "linkerd-namespace", "l", defaultNamespace, "Namespace in which Linkerd is installed [$LINKERD_NAMESPACE]")
 	RootCmd.PersistentFlags().StringVar(&kubeconfigPath, "kubeconfig", "", "Path to the kubeconfig file to use for CLI requests")
+	RootCmd.PersistentFlags().StringVar(&kubeContext, "context", "", "Context to use")
```

Comment:
I think we should somehow specify that this context applies to the kubeconfig file. I like the wording from kubectl itself:

```
$ kubectl options | grep context
      --context='': The name of the kubeconfig context to use
```

I'd change this to:

```
	RootCmd.PersistentFlags().StringVar(&kubeContext, "context", "", "Name of the kubeconfig context to use")
```